server/base/src/main/java/org/apache/accumulo/server/manager/state/MetaDataStateStore.java

```diff
@@ -62,10 +62,10 @@ public void setLocations(Collection<Assignment> assignments) throws DistributedS
     try (var tabletsMutator = ample.mutateTablets()) {
       for (Assignment assignment : assignments) {
         TabletMutator tabletMutator = tabletsMutator.mutateTablet(assignment.tablet);
-        tabletMutator.putLocation(assignment.server, LocationType.CURRENT);
+        tabletMutator.putLocation(TabletMetadata.Location.current(assignment.server));
```

Comment:
Could shorten this a bit by importing Location.  This is a stylistic comment feel free to ignore it.

```suggestion
        tabletMutator.putLocation(Location.current(assignment.server));
```

I like these static methods, I think they shorten the code w/o any loss of readability.

Reply:
Yeah I agree, it would be even nicer/cleaner if we imported Location

Reply:
Latest commit changes the imports to clean it up